### PR TITLE
Authentikos: add grandmatriarch example

### PR DESCRIPTION
**Pull request by clarketm**

### CODE REVIEW

File: authentikos/examples/authentikos-grandmatriarch-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          source.developers.google.com FALSE  {{.Now | .TimeToUnix | .Add 3600}} {{.Token}}
+        volumeMounts:
+        - name: creds
+          mountPath: /etc/creds
+          readOnly: true
+      volumes:
+      - name: creds
+        secret:
+          defaultMode: 0644
+          secretName: service-account
```
Comment (cjwagner) [inline authentikos/examples/authentikos-grandmatriarch-deployment.yaml:86]: We should use a separate secret specifically for this.

**Review by cjwagner — APPROVED** [review]

File: authentikos/examples/authentikos-grandmatriarch-deployment.yaml (hunk 1 above)
Comment (clarketm) [reply authentikos/examples/authentikos-grandmatriarch-deployment.yaml:86]: I can update this. I don’t have any preference on naming. Do you prefer a particular name (or naming convention) or is something like *authentikos-service-account* reasonable?

Comment (cjwagner) [reply authentikos/examples/authentikos-grandmatriarch-deployment.yaml:86]: That sounds great, thanks!

**Review by chases2 — APPROVED** [review]

**Comment by cjwagner** [comment]
/hold cancel
We can iterate on this if needed, but I'd like to start trying it out.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**